Packs/CertStream/Integrations/CertStream/CertStream.py

```diff
@@ -0,0 +1,354 @@
+import json
+import traceback
+from contextlib import contextmanager
+from uuid import UUID, uuid5
+
+from websockets.exceptions import InvalidStatus, ConnectionClosed
+from websockets.sync.client import connect
+
+from CommonServerPython import *  # noqa: F401
+
+VENDOR = "Kali Dog Security"
+PRODUCT = "CertStream"
+SCO_DET_ID_NAMESPACE = UUID('00abedb4-aa42-466c-9c01-fed23315a9b7')
+FETCH_SLEEP = 5
+XSOAR_TIME_FORMAT = '%Y-%m-%dT%H:%M:%S+00:00'
+
+
+def test_module(host: str):
+    # set the fetch interval to 2 seconds so we don't get timeout for the test module
+    recv_timeout = 2
+    try:
+        with websocket_connections(host) as (message_connection):
+            json.loads(message_connection.recv(timeout=recv_timeout))
+            return "ok"
+
+    except InvalidStatus as e:
+        if e.response.status_code == 401:
+            return_error("Authentication failed. Please check the Cluster ID and API key.")
+
+
+@contextmanager
+def websocket_connections(host: str):
+    demisto.info(f"Starting websocket connection to {host}")
+    url = host
+    with connect(url) as message_connection:
+        yield message_connection
+
+
+def long_running_execution_command(host: str, fetch_interval: int):
+    """ Executes to long running loop and checks if an update to the homographs is needed
+
+    Args:
+        host (str): The URL for the websocket connection
+        fetch_interval (int): The interval in minutes to check for updates to the homographs list
+    """
+
+    while True:
+        with websocket_connections(host) as (message_connection):
+            demisto.info("Connected to websocket")
+
+            while True:
+                now = datetime.now()
+                context = json.loads(demisto.getIntegrationContext()["context"])
+                last_fetch_time = datetime.strptime(context["fetch_time"], '%Y-%m-%d %H:%M:%S')
+                fetch_interval = context["list_update_interval"]
+
+                if now - last_fetch_time >= timedelta(minutes=fetch_interval):
+                    context["homographs"] = get_homographs_list(context["word_list_name"])
+                    context["fetch_time"] = datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+                    demisto.setIntegrationContext({"context": json.dumps(context)})
```

Comment:
use `set_to_integration_context_with_retries` from `commonserverpython`

Reply:
I got an error when I tried to use `set_to_integration_context_with_retries()` are you sure it needs to be changed to that?

Reply:
its better in case theres any flakiness from the server, what error did you get? Thus might be a moot point since not sure context should be used at all